.reuse/dep5

```diff
@@ -0,0 +1,6 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: firebase/*
+Copyright: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+License: MIT
+Comment: All files are part of the ENGAGE-HF application based on the Stanford Spezi Template Application project.
```

Comment:
Looking at this comment it might make sense to note that the data in the firebase directory is actually seeded data for the emulator instead of referencing the Template Application.